### ISSUE TITLE
mods

### ISSUE THREAD

**Issue by alekenby** [masterfeizz/Xash3DS#21]
is there any way to play mods or at least the expansion packs?

**Comment by VictorMX2307** [comment]
yes you just need to put the folder into de directory of xash3d example: for team fortress classic (tfc folder) put it on xash3d directory (sdmc:/xash3d/) and same for counter strike (cstrike i think this is the folder name) opposing force blue shift etc (i dint tried this yet but i think it will work)

**Comment by Xenthio** [comment]
Mods that do not use custom binaries should run fine. However, mods that do will need to be recompiled.
